llama_hub/prometheus/base.py

```diff
@@ -0,0 +1,66 @@
+from typing import List, Optional
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from datetime import datetime
+
+
+class PrometheusReader(BaseReader):
+    def __init__(self, endpoint: str, size: Optional[int] = 100):
+        try:
+            from prometheus_api_client import PrometheusConnect
+        except ImportError as err:
+            raise ImportError(
+                "`prometheus-api-client` package not found, please run `pip install prometheus-api-client`"
+            ) from err
+
+        self.prometheus_url = endpoint
+        self.prom_client = PrometheusConnect(url=self.prometheus_url)
+
+    def load_data(
+        self,
+        query: str,
+        start_time: Optional[datetime],
```

Comment:
better to make these be kwargs instead of positional args and have default values (`start_time: Optional[datetime] = None`)

Reply:
sure, let me fix it and update the PR

Reply:
is fixed now, please tell me any other issue or observation that need to be change.
Thanks